INSTALL.md

```diff
@@ -73,14 +73,14 @@ To use gflags in a Bazel project, map it in as an external dependency by editing
 your WORKSPACE file:
 
     git_repository(
-        name = "gflags_git",
+        name = "com_googlesource_code_gflags",
```

Comment:
Wouldn't `com_github_gflags` make more sense?

In any case, isn't this name just used within the user's `WORKSPACE` file ? Hence, every user can choose whatever name they like and does not conflict with other dependencies they use.